client/src/components/chat/MessagesPanel.js

```diff
@@ -34,6 +35,7 @@ export class MessagesPanel extends React.Component {
 
     render() {
         let list = [];
+        console.log(this.props.isOnline);

```

Comment:
Is this console log needed ? Lets keep the console logs to a minimum. 

Will need to clean out all other console log that print out random stuff in future.